config/locales/client.en.yml

```diff
@@ -4673,6 +4673,8 @@ en:
         problems_found: "Some advice based on your current site settings"
         new_features:
           title: "What's new"
+          previous_announcements: "You can see previous new feature announcements on "
```

Comment:
The link should be in the translation I think, IIRC this is better for translations since some languages would put this in different places depending on grammar.